eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,16 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- Ignoring the known prebuilts. -->
+    
+    <!-- Tracked in https://github.com/dotnet/source-build/issues/3437 -->
+    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/7.0.2" />
```

Comment:
The 7.0.2 app ref related package should not be added to SBRP.  These are all from fsharp targetting net7.0.  It needs to target net8.0 for source build.  This is tracked with https://github.com/dotnet/fsharp/issues/14765